karma.conf.js

```diff
@@ -1,7 +1,7 @@
 // Karma configuration
 // Generated on Fri Feb 26 2016 13:09:51 GMT+1100 (AEDT)
 "use strict";
-module.exports = function(config) {
+module.exports = function (config) {
```

Comment:
Not a fan of these changes - a space after "function" keyword

Reply:
Might be worth to check the relevant discussion...

Reply:
https://github.com/prettier/prettier/pull/3903 is the culprit, and seemingly the majority of people liked it.